### PR TITLE
AAN News Helicopter Intro Cinematic

### PR DESCRIPTION
**Pull request by bijx**
This PR improves the intro cinematic engine by creating a new cinematic type called `News Helicopter`, which borrows the orbiting logic from the `Circle Cutscene` module, but includes a shorter radius rotation, AAN news overlays, a camera zoom effect, and helicopter sfx, while still supporting the interlude titles and audio options.

image
image
image

Tested on pub zeus server.

### CODE REVIEW

**Comment by bijx** [comment]
This branch also resolves the #67 bug for create intel modules.

File: Enhanced_Zeus_Modules.sqf [hunk 1]
```diff
@@ -9359,48 +9359,102 @@ MAZ_EZM_fnc_initFunction = {
...
+					[[_target],HYPER_fnc_remoteCamera] remoteExec ["spawn", _allPlayers];
+				};
+				case "News": {
+					[[],{
+						playSoundUI ["a3\sounds_f\vehicles\air\heli_attack_01\heli_attack_01_ext_rotor.wss", 0.2];
+						sleep 7.5;
+						playSoundUI ["a3\sounds_f\vehicles\air\heli_attack_01\heli_attack_01_ext_rotor.wss", 0.2];
+					}] remoteExec ["spawn", _allPlayers];
+					private _AANParams = [
+						parseText _briefingName,
```
Comment (expung3d) [inline Enhanced_Zeus_Modules.sqf:9413]: The text utilizes [Structured Text](https://community.bistudio.com/wiki/Structured_Text). Make the text size 2:
`"<t size='2'> [TEXT HERE] </t>"`

Comment (expung3d) [reply Enhanced_Zeus_Modules.sqf:9413]: Specifically for the headline, not the rolling text.

**Review by expung3d — CHANGES_REQUESTED** [review]

**Comment by expung3d** [comment]
Also, another idea. Maybe have the textiles be just for the Flyby mode. Otherwise, make it the rolling text for the AAN display. It looks kinda odd having the textiles AND the AAN display. If it were the news the textiles would be on the roller.

**Comment by expung3d** [comment]
Now that it uses the utility functions you don't need to disable the simulation of vehicles yourself. It does that in the `MAZ_EZM_fnc_enterCinematicCamera` function.

**Review by expung3d — APPROVED** [review]

**Pull request merged**